profile/README.md

```diff
@@ -7,13 +7,43 @@ BDK is based on the powerful [`rust-bitcoin`](https://github.com/rust-bitcoin/ru
 
 Learn the simplest way to integrate Bitcoin wallet features into any application at [bitcoindevkit.org](https://bitcoindevkit.org/).
 
-### 🔨 Contributing to the ecosystem
-Our team maintains [bdk](https://github.com/bitcoindevkit/bdk) which is the core library, while also maintaining a number of other related open source projects like:
- - [BDK CLI](https://github.com/bitcoindevkit/bdk-cli) - A CLI wallet library and REPL tool to demo and test the BDK library
- - [BDK FFI](https://github.com/bitcoindevkit/bdk-ffi) - A foreign language bindings generator for BDK (experimental)
- - [BDK Documentation](https://bitcoindevkit.org/) - Our open source documentation
+### Our core libraries
+
+The core libraries are developed and maintained collectively by the Bitcoin Dev Kit devs. The following table outlines those libraries as well as an official maintainer for each of them.
+
+| Library                 | Repository                                                                    | Lead Maintainer    |
+| ----------------------- | ----------------------------------------------------------------------------- | ------------------ |
+| `bdk_wallet`            | [bdk](https://github.com/bitcoindevkit/bdk)                                   | [ValuedMammal]     |
+| `bdk_chain`             | [bdk](https://github.com/bitcoindevkit/bdk)                                   | [LagginTimes]      |
+| `bdk_core`              | [bdk](https://github.com/bitcoindevkit/bdk)                                   |                    |
+| `bdk_esplora`           | [bdk](https://github.com/bitcoindevkit/bdk)                                   |                    |
+| `bdk_electrum`          | [bdk](https://github.com/bitcoindevkit/bdk)                                   |                    |
+| `bdk_bitcoind_rpc`      | [bdk](https://github.com/bitcoindevkit/bdk)                                   |                    |
```

Comment:
Even though kyoto is still technically "experimental" I'd like to list it here to give it visibility
```suggestion
| `bdk_bitcoind_rpc`      | [bdk](https://github.com/bitcoindevkit/bdk)                                   |                    |
| `bdk-kyoto`                  | [bdk-kyoto](https://github.com/bitcoindevkit/bdk-kyoto)              | [rustaceanrob] |
```